misc/versionList.md

```diff
@@ -3,6 +3,23 @@
 In addition to various minor changes, the following was
 changed for the different versions of ꟻLIP:
 
+# Version 1.4 (commit nnnnnnn)
+- Changed the Python version of ꟻLIP so that it leverages the C++ code through [pybind11](https://github.com/pybind/pybind11).
+	- Results (only evaluation, not including file load/save, etc):
+		- 16-26x faster for LDR/HDR CPU (depends on CPU setup).
+		- Timings for 1920x1080 images:
+			- Python/LDR: 91 ms
```

Comment:
For absolute numbers, e.g., 91 ms, it would be nice if you could add what kind of CPU etc you used. Otherwise, it does not mean much.

Reply:
Pushed update to performance numbers, including information about the CPU used. This information was also added to the performance numbers stated for the FLIP 1.2 update.